src/components/Editor/MediaUploader/constants.js

```diff
@@ -1,34 +1,36 @@
 import { globalProps } from "neetocommons/initializers";
 
+import { DIRECT_UPLOAD_ENDPOINT } from "src/common/constants";
+
 const MAX_VIDEO_SIZE = 100 * 1024 * 1024; // 100 MB
 
 export const FILE_SIZE_UNITS = ["B", "KB", "MB", "GB"];
 
 export const ALLOWED_IMAGE_TYPES = [".jpg", ".jpeg", ".png", ".gif"];
 export const ALLOWED_VIDEO_TYPES = [".mp4", ".mov", ".avi", ".mkv"];
 
-export const DEFAULT_IMAGE_UPPY_CONFIG = {
+export const DEFAULT_IMAGE_UPLOAD_CONFIG = {
   autoProceed: false,
   allowMultipleUploads: true,
```

Comment:
`autoProceed` and `allowMultipleUploads` are pretty much useless config since `@uppy` is removed. You can remove them.